{{cookiecutter.package_name}}/setup.py

```diff
@@ -43,6 +43,7 @@
         long_description = f.read()
 
     install_requirements = [
+        "vivarium_build_utils>=1.1.0,<{{cookiecutter.vivarium_build_utils_next_major_version}}",
```

Comment:
is this the most sensible? Or we could just request the user to input the entire pin (rather than hard-coding the >=1.1.0 part)?